src-docs/src/views/combo_box/combo_box.js

```diff
@@ -74,7 +74,8 @@ export default () => {
     /* DisplayToggles wrapper for Docs only */
     <DisplayToggles canDisabled={false} canReadOnly={false} canIsDisabled>
       <EuiComboBox
-        placeholder="Select or create options"
+        aria-label="Accessible screen reader label"
+        placeholder="Select or create options for"
```

Comment:
Was this extra "for" intentional?
```suggestion
        placeholder="Select or create options"
```

Reply:
It wasn't. 😬 I must have been switching programs too quickly.